reana_client/cli/workflow.py

```diff
@@ -128,7 +117,7 @@ def workflow_workflows(
     access_token,
     show_all,
     verbose,
-    block_size,
+    unit_format,
```

Comment:
Now I see, regarding https://github.com/reanahub/reana-client/pull/435#discussion_r510811001, this was the reason. Because human readable at code level is not a boolean, but text (`["human_readable"|"raw"]`), I used `unit_format`. Then to be consistent I renamed the `human_readable_option` to `unit_format_option`. Let me know what you think, I am fine even changing this one parameter (here and elsewhere) to `human_readable` even if it is a bit misleading in my opinion.

Reply:
Ah ok, it makes more sense then. Still tricky to figure out what `unit_format` means without checking the code.
IMO you can leave `unit_format`, not a strong opinion anymore.

Reply:
I've finally tried to have a compromise taking into account our conversation and went fro `human_readable_or_raw_option` and the parameter `human_readable_or_raw`. Still not the best but at least it is: explicit and consistent, how pretty it is is arguable and if we want to change it we can do it outside this PR, thanks for the valuable input :).